README.md

```diff
@@ -42,46 +42,53 @@ $ ./nav -f conf.json -s kernel_init
 The following is the command line switches list from the nav help.
 ```
 $ ./nav -h
-Command Help
-App Name: nav
-Descr: kernel symbol navigator
-	-j	<v>	Force Json output with subsystems data
-	-s	<v>	Specifies symbol
-	-i	<v>	Specifies instance
-	-f	<v>	Specifies config file
-	-e	<v>	Forces to use a specified DB Driver (i.e. postgres, mysql or sqlite3)
-	-d	<v>	Forces to use a specified DB DSN
-	-m	<v>	Sets display mode 2=subsystems,1=all
-	-h		This Help
+Nav - kernel symbol navigator
+
+Usage:
+  nav [FLAGS]
+
+Flags:
+  -f, --config config              path to config file
+  -s, --symbol symbol              name of the symbol to start the navigation from
+  -j, --output-type type           type of output: graphOnly, jsonOutputPlain, jsonOutputB64 or jsonOutputGZB64 (default "graphOnly")
+  -x, --max-depth number           max number of levels in call flow exploration (0=No limit)
+  -m, --mode mode                  mode of plotting: 1=Symbols, 2=Subsystems, 3=Subsystems with labels, 4=Target subsystem isolation (default 2)
+  -b, --excluded-before symbols    list of symbols to exclude before the target symbol
+  -a, --excluded-after symbols     list of symbols to exclude after the target symbol
+  -t, --target-subsys subsystems   list of subsystems to include in the output
+  -e, --db-driver driver           database driver: mysql, postgres or sqlite3 (default "postgres")
+  -d, --DBDSN DSN                  database DSN (format: "host=dbhost.com port=5432 user=username password=<password> dbname=kernel_bin sslmode=disable")
```

Comment:
Here we give a format, but later we say "DSN in the engine specific format", which seems more correct. I think we need to give examples for all DB engines we support.

Reply:
Agreed, providing examples for all the supported database engines could be helpful for further clarity. 
As for DSN documentation, I have already started documenting it in the kern_bin_db repository's README[1]. 
It might be worth repeating the same information in the nav README for easier reference. 
Please note that the documentation is not exhaustive as I haven't had the chance to test all the DSN, but it still provides useful indications, including for the untested sqlite. 

[1]. https://github.com/AC-FuSa-Tools/kern_bin_db-/blob/main/README.md"

Reply:
Addressing this in the following commit, therefore I'm marking this as resolved.